swingset/src/main/java/com/nqadmin/swingset/SSComboBox.java

```diff
@@ -76,9 +76,9 @@
  * are the correct techniques.
  * There are a variety of method to change the current combobox item
  * such as
- * {@link #setSelectedMapping(java.lang.Object) setSelectedMapping(Integer)}.
+ * {@link #setSelectedMapping(Integer)} / {@link #setSelectedItem(Object)}.
```

Comment:
It might not be a good idea to put `setSelectedItem(Object)` here. Users of this class won't have an easy way to create an SSListItem.